components/card/cards.css

```diff
@@ -0,0 +1,194 @@
+.card-vertical,
+.card-about,
+.card-img{
+    width:23rem;
+}
+.card-container-vertical{
+    display: flex;
+    flex-direction: row;
+}
+.iframe-card{
+    margin-left:10rem;
+    margin-top:1.5rem;
+}
+
+/* vertical card */
+.card-vertical {
+    background-color: white;
+    box-sizing: border-box;
+    box-shadow: 1px 3px 3px #e5e5e5;
+    border-radius: 0.5rem;
+    width: 23rem;
+    height: 38rem;
+    display: flex;
+    flex-direction: column;
+    align-items: left;
+    margin-bottom:3rem;
+  }
+ 
+.card-img {
+    height: 23rem;
+    width:23rem;
+    border-radius: 0.5rem 0.5rem 0 0;
+    /* margin-top:0; */
```

Comment:
your images are squeezed. Use  `object-fit:cover` inside img class.

Reply:
thanks for the suggestion. It now looks  much better.